package.json

```diff
@@ -17,6 +17,7 @@
     "packages/rln",
     "packages/tests",
     "packages/browser-tests",
```

Comment:
if `browser-tests` depends on `headless-tests` - then they should be defined in different order in the array

also, about the naming, maybe it is better to rename:
- `browser-tests` -> `browser-container`;
- `headless-tests` -> `browser-tests`;

That way we define that one is container and another is test cases.

Reply:
let's do this separate because it will make review terrible

Reply:
> @adklempner why? it seems that just lines needs to be swapped

I meant the renaming, you are correct about the order